### ISSUE TITLE
[Train SCNN-TensorFlow] : <class 'tensorflow.python.framework.errors_impl.CancelledError'>, Enqueue operation was cancelled.

### ISSUE THREAD

**Issue by earlysleepearlyup** [cardwing/Codes-for-Lane-Detection#175]
When I train the SCNN-TensorFlow, the error trun out.
This is the error:

> Epoch: 90094 loss_ins= 0.003494 (0.003464) loss_ext= 0.000006 (0.000006) accuracy= 0.988932 (0.989564) accuracy_back= 0.997696 (0.997700) mean_time= 0.401934s 
> Epoch: 90095 loss_ins= 0.003441 (0.003463) loss_ext= 0.000006 (0.000006) accuracy= 0.990375 (0.989573) accuracy_back= 0.997693 (0.997699) mean_time= 0.402190s 
> Epoch: 90096 loss_ins= 0.003497 (0.003464) loss_ext= 0.000006 (0.000006) accuracy= 0.990354 (0.989581) accuracy_back= 0.997650 (0.997699) mean_time= 0.402138s 
> Epoch: 90097 loss_ins= 0.003340 (0.003462) loss_ext= 0.000006 (0.000006) accuracy= 0.988602 (0.989571) accuracy_back= 0.997772 (0.997700) mean_time= 0.402131s 
> Epoch: 90098 loss_ins= 0.003446 (0.003462) loss_ext= 0.000006 (0.000006) accuracy= 0.989145 (0.989567) accuracy_back= 0.997752 (0.997700) mean_time= 0.402337s 
> Epoch: 90099 loss_ins= 0.003515 (0.003463) loss_ext= 0.000006 (0.000006) accuracy= 0.988700 (0.989558) accuracy_back= 0.997706 (0.997700) mean_time= 0.402565s 
> Epoch: 90100 loss_ins= 0.003551 (0.003464) loss_ext= 0.000006 (0.000006) accuracy= 0.988906 (0.989551) accuracy_back= 0.997696 (0.997700) mean_time= 0.402450s 
> 2019-10-25 04:49:09.973449: W tensorflow/core/kernels/queue_base.cc:277] _1_data_augmentation_1/input_producer/input_producer: Skipping cancelled enqueue attempt with queue not closed
> 2019-10-25 04:49:09.973524: W tensorflow/core/kernels/queue_base.cc:285] _3_batch/fifo_queue: Skipping cancelled dequeue attempt with queue not closed
> 2019-10-25 04:49:09.973571: W tensorflow/core/kernels/queue_base.cc:277] _0_data_augmentation/input_producer/input_producer: Skipping cancelled enqueue attempt with queue not closed
> 2019-10-25 04:49:09.973608: W tensorflow/core/kernels/queue_base.cc:277] _4_prefetch_queue_1/fifo_queue: Skipping cancelled enqueue attempt with queue not closed
> 2019-10-25 04:49:09.973652: W tensorflow/core/kernels/queue_base.cc:277] _5_batch_1/fifo_queue: Skipping cancelled enqueue attempt with queue not closed
> I1025 04:49:09.974268 140578817820416 coordinator.py:224] Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.CancelledError'>, Enqueue operation was cancelled
> 	 [[{{node batch_1/fifo_queue_enqueue}}]]

I have googled this soultion:
```
coord = tf.train.Coordinator()
threads = tf.train.start_queue_runners(sess=sess, coord=coord)
```
but it not work.

**Comment by cardwing** [comment]
I have not met this problem before. You need to debug by yourself.
